README.md

```diff
@@ -47,20 +47,20 @@ The metrics collected by Model Analyzer are listed below:
 
 ## Documentation
 
-Model Analyzer runs as a command line interface (CLI), standalone Docker container, or Helm chart. Documentation for each is provided in the [docs] folder.
+Triton Memory Analyzer runs as a command line interface (CLI), standalone Docker container, or Helm chart. Documentation for each is provided in the [docs] folder.
 
 ## Quick Start
 
-The [docs] folder hosts all information about building Model Analyzer from source.  
-If you want to use Model Analyzer right away, there are two options:
+The [docs] folder hosts all information about building Triton Memory Analyzer from source.
+If you want to use Triton Memory Analyzer right away, there are two options:
 
 ##### You have supported hardware and drivers, [NVIDIA-Docker], and [.NET SDK].
 
 ```
-git clone https://github.com/NVIDIA/model-analyzer
-cd src
+git clone https://github.com/triton-inference-server/model_analyzer.git
+cd model_analyzer
```

Comment:
Small fix: cd model_analyzer/src

(Sorry for any confusion. Good to then merge. Thanks for your updates!)

Reply:
updated